site/docs/hosting/deno-deploy.md

```diff
@@ -59,42 +59,44 @@ Here, we are using the bot token (`/<bot token>`).
 
    > It is recommended that you have a single stable branch and you do your testing stuff in other branches, so that you won't get some unexpected things happen.
 
-3. Visit your [Deno Deploy dashboard](https://dash.deno.com/projects).
-4. Click on "New Project", and go to the "Deploy from GitHub repository" section.
+3. Visit your [Deno Deploy dashboard](https://dash.deno.com/account/overview).
+4. Click on "New Project".
 5. Install the GitHub app on your account or organization, and choose your repository.
-6. Select the branch you want to deploy, and then choose your `mod.ts` file to be deployed.
+6. Select the branch you want to deploy.
+7. Select the entrypoint file `main.ts`, and click "Deploy Project" to deploy.
 
 ### Method 2: With `deployctl`
 
-> This is a method for more advanced users. It allows you to deploy the project via the command line or Github Actions.
+> This is a method for more advanced users or if you don't want to upload your code to GitHub.
+> It allows you to deploy the project via the command line or Github Actions.
 
-1. Visit your [Deno Deploy dashboard](https://dash.deno.com/projects).
-2. Click "New Project", and then "Empty Project".
-3. Install [`deployctl`](https://github.com/denoland/deployctl).
-4. [Create an access token](https://dash.deno.com/account#access-tokens).
-5. Run the following command:
+1. Install [`deployctl`](https://github.com/denoland/deployctl).
+2. Create an access token from the "Access Tokens" section in [account settings](https://dash.deno.com/account).
```

Comment:
The `access-token` anchor was removed due to the page's lazy loading behavior, so that link-checker doesn't have to be configured to check for the anchor. I think the information is good enough. But if it needs to be reverted, let me know, I can adjust link-checker to ignore this anchor.